set.mm

```diff
@@ -116418,6 +116471,13 @@ used to convert hypothesis of the inference (deduction) form of this
     wceq ) AEFBEFGZCEFDEFGZGABHIZJZCDHIZJZRZUAUCRZBAHIZDCHIZRSUAEFUCEFUEUFKTABL
     CDLUAUCMNSTUBUGUDUHABOCDOPQ $.
 
+  $( Two complex numbers which add up to zero are each other's opposites.
```

Comment:
Two complex numbers add up to zero iff they are each other's opposites.

Reply:
Fixed